product-features/self-service-farming.md

```diff
@@ -0,0 +1,114 @@
+3. The total amount of rewards for the entire farm lifecycle must be deposited at the time of farm creation.
+4. A "gathering" period occurs between the creation of the farm and the start of the first emission cycle. Farmers who stake their LP tokens during this period will be eligible to receive the farming rewards associated with the first emission cycle.
+
+## 📝 Procedure
+
+### Step 1
+
+Go to the [Farms page](https://app.alexlab.co/farm) and click on the "Create" button.
+
+<figure><img src="../.gitbook/assets/self-service-farming/create-farm-button.png" alt="" width="563"><figcaption></figcaption></figure>
+
+In this guide we are assuming that you already have a pool, so select the "Creating a new farm" option and click "Continue".
+
+<figure><img src="../.gitbook/assets/self-service-farming/create-type-new-farm.png" alt="" width="375"><figcaption></figcaption></figure>
+
+### Step 2
+
+Select a pool from the ones you've created.
+
+<figure><img src="../.gitbook/assets/self-service-farming/step-1-select-a-pool.png" alt=""><figcaption></figcaption></figure>
+
+### Step 3
+
+Enter the total amount of rewards that will be distributed in your farm. This amount is deposited at farm creation. Also, remember that in Self-Service Farming, the reward token has to be the same as the listing token.
+
+<figure><img src="../.gitbook/assets/self-service-farming/enter-total-rewards-amount.png" alt=""><figcaption></figcaption></figure>
+
+### Step 4
+
+Select the **Reward Cycle Length**, which is the number of cycles in which your farm will be active and distributing rewards. This number, along with the total farming rewards, determines the **Est. Farming Rewards Per Cycle**.
+
+At this point, you will also be able to see the number of the cycles in which your farm will be officially open, displayed as **Farm Opening Cycle**. The farm will be created instantly once the farm creation transaction is confirmed. However, the **Emission Start Cycle** will be the next upcoming cycle.
+
+The time gap between the farm creation and the start of the first emission cycle is the so-called "gathering" period, during which users can begin staking their LP tokens in the farm.
+
+{% hint style="danger" %}
+**Caution:** To maximize the gathering period, it is advisable to create your farm at the start of a new farming cycle.
+{% endhint %}
+
+<figure><img src="../.gitbook/assets/self-service-farming/step-2-add-farming-rewards.png" alt=""><figcaption></figcaption></figure>
+
+#### Example
+
+From the screenshot above, the **Total Farming Rewards** is `15,000 DOGGY`, and **Reward Cycle Length** is `10`, meaning that `15,000 DOGGY / 10 = 1,500 DOGGY` will be the **Farming Rewards Per Cycle**.
+
+If the user submits, the farm will be created at Cycle #80, leaving 33 blocks for the gathering period (approximately 5 hours and 29 minutes). In these cases, it is recommended to wait for the next cycle, as inidicated in the alert box.
```

Comment:
Add "This prevents LP tokens from being committed to the farm not in time for the next reward distribution cycle."